cdap-runtime-spi/src/main/java/io/cdap/cdap/runtime/spi/runtimejob/RuntimeJobDetail.java

```diff
@@ -24,8 +24,8 @@
  * Represents runtime job details.
  */
 public class RuntimeJobDetail {
-  private final ProgramRunInfo runInfo;
-  private final RuntimeJobStatus status;
+  protected final ProgramRunInfo runInfo;
```

Comment:
Don't change it to protected. Use the getter methods in the sub-class to access the field instead.

Reply:
Done.